### PR TITLE
Lab-3

### PR DESCRIPTION
**Pull request by aekrops**
- [x] Home view from the @Denys-Doskochynskiy second lab 
 
#### Original:
image
 
#### Portrait: 
image
image
image

#### Landscape:
image
image
image

***

- [x] Social view (@aekrops)
#### Original:
image

#### Portrait:
image
image

#### Landscape:
image
image

***

- [x] Profile view (@Denys-Doskochynskiy)
### Original:
image

#### Portrait:
image
image

#### Landscape:
image
image
image

***

- [x] Settings view (@damoklov)
#### Original:
image

#### Portrait:
image
image

#### Landscape:
image
image
image

### CODE REVIEW

File: lib/social_view/element/friends_list.dart [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    var phoneSize = MediaQuery.of(context).size;
+    return Container(
+        width: phoneSize.width,
+        margin: const EdgeInsets.only(left: 10, top: 5),
+        child: Align(
+            alignment: Alignment.centerLeft,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: const [
+                StyledText("Online", 25.0, Color(0xFFFFFFFF), FontWeight.w400),
```
Comment (igor-leshkevych) [inline lib/social_view/element/friends_list.dart:21]: 0xfff.. is just white. It's better to define some object with common colors (`class Theme { Color textColor = Colors.white; }`) and use them everywhere

File: lib/object/friends_obj.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Friend {
+  final String nickname;
+  final String imgLink;
+  final bool onlineStatus;
+
+  Friend(this.nickname, this.imgLink, this.onlineStatus);
+}
+
+final friends = [
```
Comment (igor-leshkevych) [inline lib/object/friends_obj.dart:9]: 👍

File: lib/object/games_obj.dart [hunk 3]
```diff
@@ -2,26 +2,41 @@ class GameContent {
...
   String icon;
+  String score;
+  String percentAchieve;
+  String winnerCup;
 
   GameContent(
       {this.title = 'Need for Speed Payback',
       this.description = 'Jump back in',
-      this.icon = 'assets/images/img_1.png'});
+      this.icon = 'assets/images/img_1.png',
```
Comment (igor-leshkevych) [inline lib/object/games_obj.dart:12]: I like the consistency of image naming. The only improvement I'd suggest is making those name more informative, so you can figure out what this image is by it's name, for example: `assets/images/game_1_nfs.png` and so on

**Review by igor-leshkevych — APPROVED** [review]
App looks good, you did really good job in terms of replicating the original UI, organizing the navigation between screens and structuring your code, too.

Now regarding your PRs. 
At our classes we're all learning: you're coding with your friends (not some guys you barely know, like in real companies), you can afford to try different things, make mistakes, look silly, and so on, and noone will judge you for that. 
In real life though, you might be less lucky with your peers, and things that worked here (empty PR descriptions, merging without approvals, skipping the code review part), won't work with them. 
That's why I would recommend practicing those things here, in a friendly environment first, so when you find yourself in a more demanding environment, you're well prepared.

**Review by damoklov — COMMENTED** [review]
Guys, I would suggest the following:
1. Use constant colors instead of 0xFF.. 
2. Rename global package names to smth more consistent
3. Use image naming convention e.g. "screen_purpose_number"

**Pull request merged**